assets/src/edit-story/components/canvas/useInsertElement.js

```diff
@@ -75,9 +75,12 @@ function useInsertElement() {
         setTimeout(() => {
           const videoEl = document.getElementById(`video-${id}`);
           if (videoEl) {
-            videoEl.play();
+            const playPromise = videoEl.play();
+            if (playPromise) {
+              playPromise.catch(() => {});
+            }
```

Comment:
I think all browsers, that we care about, return a promise here, so this can just be:

```suggestion
            videoEl.play().catch(() => {});
```

Reply:
(similarly in the other copies of these lines below)

Reply:
Fair enough 👍